manual/src/cmds/intf-c.etex

```diff
@@ -573,28 +573,25 @@ default is the boxed representation.
 
 \subsection{ss:c-arrays}{Arrays}
 
-Arrays of integers and pointers are represented like tuples,
+Arrays of integers and pointers are represented like tuples and records,
 that is, as pointers to blocks tagged~0.  They are accessed with the
 "Field" macro for reading and the "caml_modify" function for writing.
 
-Arrays of floating-point numbers (type "float array")
-have a special, unboxed, more efficient representation.
-These arrays are represented by pointers to blocks with tag
-"Double_array_tag".  They should be accessed with the "Double_array_field"
-and "Store_double_array_field" macros.
-
-The special representation of "float array" necessitates an extra runtime check
-in most polymorphic array operations. To avoid this penalty, a dedicated type
-"floatarray" is also available which enjoys the special representation
-unconditionally. Values of type "floatarray", as well as records consisting
-solely of floats, should be accessed using the "Double_flat_field" and
+Values of type "floatarray" (as manipulated by the "Float.Array"
+module), as well as records whose declaration contains only float
+fields, use an efficient unboxed representation: blocks whose content
+consist of raw double values, which are not themselves valid OCaml
+values. They should be accessed using the "Double_flat_field" and
 "Store_double_flat_field" macros.
```

Comment:
Nitpicking over my own suggestion: this paragraph should mention that the unboxed representation uses the tag Double_array_tag. Maybe instead of `blocks whose content ...` it could be `blocks with tag Double_array_tag, whose content ...`.

Reply:
Makes sense, fixed, thanks.